book/index.md

```diff
@@ -24,13 +24,12 @@ have reached the limits of their capabilities.
 
 Whatever the case, we’re confident that the usefulness of algorithms and
 data structures is proportional to the depth at which you understand it.
-Those who understand trees in depth are more likely to identify tree
-shapes in their own data, unlocking superior solutions to their
-problems. Those who count big O analysis as a tool in their toolkits not
-only have a better vocabulary for comparing alternative strategies, they
-are also more generally cognizant of the trade-offs we implicitly make
-every day. Those who see recursion everywhere see recursion everywhere...
-:)
+Those who study trees are more likely to identify tree shapes in their own
+data, unlocking superior solutions to their problems. Those who have big O
+analysis in their toolkits not only have a better vocabulary for comparing
+alternative strategies, they are also more generally cognizant of the
+trade-offs we implicitly make every day. Those who see recursion everywhere see
+recursion everywhere... :)
```

Comment:
Forgot while I was working, but maybe it's finally time to remove the smiley... :sob: